src/main/java/scene/Scene.java

```diff
@@ -59,9 +59,22 @@
  *         public void awake() {
  *             camera = new Camera();
  *
+ *             // this is the player //
```

Comment:
In this case, the comment is not necessary... the name of the variable explains the purpose of the object.
![image](https://user-images.githubusercontent.com/32278830/140093229-8f4fd3a3-d24d-4806-a31e-a49367bc3e54.png)